frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -708,14 +703,19 @@ export const MarkdownView = (props: {
   );
 };
 
-export const CRDCard: React.FC<CRDCardProps> = (props) => {
-  const { csv, crd, canCreate, required = false } = props;
+export const CRDCard: React.FC<CRDCardProps> = ({ csv, crd, required, ...rest }) => {
   const reference = referenceForProvidedAPI(crd);
-  const model = modelFor(reference);
-  const createRoute = () =>
-    `/k8s/ns/${csv.metadata.namespace}/${ClusterServiceVersionModel.plural}/${csv.metadata.name}/${reference}/~new`;
+  const [model] = useK8sModel(reference);
+  const canCreate = rest.canCreate ?? model?.verbs?.includes?.('create');
+  const createRoute = React.useMemo(
+    () =>
+      csv
+        ? `/k8s/ns/${csv.metadata.namespace}/${ClusterServiceVersionModel.plural}/${csv.metadata.name}/${reference}/~new`
+        : null,
+    [csv, reference],
+  );
 
-  return (
+  return model ? (
```

Comment:
We're now purposely showing cards for resources that don't yet have a model. See #7393